docs/.readthedocs.yaml

```diff
@@ -10,4 +10,7 @@ sphinx:
 
 python:
   install:
-    - requirements: docs/requirements.txt
\ No newline at end of file
+    - requirements: docs/requirements.txt
+
+formats:
+  - pdf
```

Comment:
How is it supposed to work ?
make latexpdf returns an error for me
! LaTeX Error: Unknown graphics extension: .svg.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.86 ...inxincludegraphics{{logo_lfe_powsybl}.svg}